packages/commonwealth/client/scripts/views/pages/CommunityManagement/Topics/CreateTopicsSection/CreateTopicSection.tsx

```diff
@@ -0,0 +1,181 @@
+import { pluralizeWithoutNumberPrefix } from 'client/scripts/helpers';
+import { useCommonNavigate } from 'client/scripts/navigation/helpers';
+import app from 'client/scripts/state';
+import {
+  useCreateTopicMutation,
+  useFetchTopicsQuery,
+} from 'client/scripts/state/api/topics';
+import { CWCheckbox } from 'client/scripts/views/components/component_kit/cw_checkbox';
+import { CWText } from 'client/scripts/views/components/component_kit/cw_text';
+import { CWValidationText } from 'client/scripts/views/components/component_kit/cw_validation_text';
+import { CWTextInput } from 'client/scripts/views/components/component_kit/new_designs/CWTextInput';
+import { CWButton } from 'client/scripts/views/components/component_kit/new_designs/cw_button';
+import {
+  ReactQuillEditor,
+  createDeltaFromText,
+  getTextFromDelta,
+} from 'client/scripts/views/components/react_quill_editor';
+import { serializeDelta } from 'client/scripts/views/components/react_quill_editor/utils';
+import type { DeltaStatic } from 'quill';
+import React, { useEffect } from 'react';
+import './CreateTopicSection.scss';
+
+export const CreateTopicSection = () => {
+  const { mutateAsync: createTopic } = useCreateTopicMutation();
+  const navigate = useCommonNavigate();
+  const { data: topics } = useFetchTopicsQuery({
+    communityId: app.activeChainId(),
+  });
+
+  const [errorMsg, setErrorMsg] = React.useState<string | null>(null);
```

Comment:
let's not use `React` prefix, just `useState`

Reply:
Done!